experimental/components/src/personnames/formatter.rs

```diff
@@ -2,11 +2,11 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
```

Comment:
I just noticed that you are moving personnames; there is a big PR which has been open for some months which I'm perpetually halfway through reviewing. (It's also there if you want to review it)

Reply:
It should be a clean-enough move, and if it's been open for a few months it's not something I want to block on.

Reply:
I'm gonna merge this PR and then update the personnames PR (#4050)